manifests/06-metricsservice.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: samples-operator-tls
```

Comment:
is there a non-alpha version of this api we can use?

Reply:
According to https://github.com/openshift/service-ca-operator there is now `service.beta.openshift.io/serving-cert-secret-name`

So yeah it has at least gone to beta now.

I will update.